### PR TITLE
Bump gcc to 11.1.0, add strchr

### PR DESCRIPTION
**Pull request by ptia**

### CODE REVIEW

**Comment by ljanyst** [comment]
These are unrelated changes. Please split into two commits.

File: libruncxx/string.cc [hunk 1]
```diff
@@ -45,3 +45,12 @@ extern "C" void *memset(void *s, int c, size_t n) {
   }
   return s;
 }
+
+extern "C" char *strchr(const char *s, int c) {
```
Comment (ljanyst) [inline libruncxx/string.cc:49]: Why is this needed?

**Review by jfrohnhofen — APPROVED** [review]

File: libruncxx/string.cc (hunk 1 above)
Comment (ptia) [reply libruncxx/string.cc:49]: gcc_11.1's unwind-dw2.c for aarch64 needs it. So it fails to link without. Weirdly enough the older gcc was fine without, and x86_64 is not affected.
